xla/stream_executor/build_defs.bzl

```diff
@@ -20,9 +20,6 @@ def tf_additional_cudnn_plugin_copts():
 def if_gpu_is_configured(x):
     return if_cuda_is_configured(x) + if_rocm_is_configured(x)
 
-def if_cuda_or_rocm(x):
```

Comment:
I think we still need that macro, it is used for example for some targets in the tools directory.

Reply:
Thanks for notice! Added it back.